test/langtools/tools/javac/reflect/ConstantsTest.java

```diff
@@ -215,4 +215,16 @@ void test12() {
     void test13() {
         Class<?> s = float.class;
     }
+
+    @IR("""
```

Comment:
Thanks - the test looks good. Is it worth it to maybe also try `String[][]` and `String[][][]` ?

Reply:
And, also a primitive array?

Reply:
Sure, would you prefer it in the same test case or should I create addtional methods?

Reply:
I recommend a separate test like the others, then the tests are focused and don't include additional operations to package up multiple values and return them.

Reply:
I added some tests with primitive arrays and more dimensions.